Kinvey.Core/Auth/User.cs

```diff
@@ -175,14 +175,31 @@ public bool Disabled
 			}
 		}
 
-		#endregion
+        /// <summary>
+		/// A boolean flag to know whether the current user is active.
+		/// </summary>
+        [JsonIgnore]
+        public bool Active
+        {
+            get
+            {
+                if (KinveyClient.IsUserLoggedIn() &&
+                KinveyClient.ActiveUser.Id == this.Id)
```

Comment:
@eugeninnowise, that's just a suggestion, it does not have to be implemented, but why not return the result of the condition for the if statement itself?